app/views/content_items/_body_with_related_links.html.erb

```diff
@@ -24,3 +24,8 @@
 
   <%= render 'shared/sidebar_navigation' %>
 </div>
+
+<% if @content_item.has_policy_and_engagement_links? %>
+  <%= render "govuk_publishing_components/components/highlight_boxes", {
+      items: @content_item.policy_and_engagement_links_content } %>
+<% end %>
```

Comment:
Missing newline at end of file 🙂 You can probably configure your editor to do this automatically.